### DISCUSSION TITLE
Every type is Any

### DISCUSSION THREAD

**Discussion by jvitormelo** [seleniumbase/SeleniumBase#2928] [Q&A] [answered]
Hi! I'm new to Python, and every function/Class I use from SeleniumBase has the Any type.

![image](https://github.com/user-attachments/assets/f18d755c-4c1f-450d-b929-0761c3c80c0b)

Is this the intended way? Or I'm missing some extension, others functions  like the sleep from time has type
![image](https://github.com/user-attachments/assets/6f1e310e-d986-4482-aa58-0acb7612b8cd)

**Comment by mdmintz** [comment, marked as answer]
Hello. Many of the args aren't given a `type`, as the `type` isn't required.

In the case of the `Driver()`, there are many args. See: https://github.com/seleniumbase/SeleniumBase/blob/f65a302a0c657de1c7944004f09efabafb51c4dc/seleniumbase/plugins/driver_manager.py#L66-L133
